File: stkpush/src/main/java/com/freddygenicho/mpesa/stkpush/api/response/STKQueryResponse.java
package com.freddygenicho.mpesa.stkpush.api.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * @author Fredrick Ochieng on 02/02/2018.
 */

public class STKQueryResponse {

    @SerializedName("ResponseCode")
    @Expose
    private String responseCode;
    @SerializedName("ResponseDescription")
    @Expose
    private String responseDescription;
    @SerializedName("MerchantRequestID")
    @Expose
    private String merchantRequestID;
    @SerializedName("CheckoutRequestID")
    @Expose
    private String checkoutRequestID;
    @SerializedName("ResultCode")
    @Expose
    private String resultCode;
    @SerializedName("ResultDesc")
    @Expose
    private String resultDesc;

    public String getResponseCode() {
        return responseCode;
    }

    public void setResponseCode(String responseCode) {
        this.responseCode = responseCode;
    }

    public String getResponseDescription() {
        return responseDescription;
    }

    public void setResponseDescription(String responseDescription) {
        this.responseDescription = responseDescription;
    }

    public String getMerchantRequestID() {
        return merchantRequestID;
    }

    public void setMerchantRequestID(String merchantRequestID) {
        this.merchantRequestID = merchantRequestID;
    }

    public String getCheckoutRequestID() {
        return checkoutRequestID;
    }

    public void setCheckoutRequestID(String checkoutRequestID) {
        this.checkoutRequestID = checkoutRequestID;
    }

    public String getResultCode() {
        return resultCode;
    }

    public void setResultCode(String resultCode) {
        this.resultCode = resultCode;
    }

    public String getResultDesc() {
        return resultDesc;
    }

    public void setResultDesc(String resultDesc) {
        this.resultDesc = resultDesc;
    }

}

File: stkpush/src/main/java/com/freddygenicho/mpesa/stkpush/api/response/STKQueryResponse.kt
package com.freddygenicho.mpesa.stkpush.api.response

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

/**
 * @author Fredrick Ochieng on 02/02/2018.
 */
class STKQueryResponse {
    @SerializedName("ResponseCode")
    @Expose
    var responseCode: String? = null

    @SerializedName("ResponseDescription")
    @Expose
    var responseDescription: String? = null

    @SerializedName("MerchantRequestID")
    @Expose
    var merchantRequestID: String? = null

    @SerializedName("CheckoutRequestID")
    @Expose
    var checkoutRequestID: String? = null

    @SerializedName("ResultCode")
    @Expose
    var resultCode: String? = null

    @SerializedName("ResultDesc")
    @Expose
    var resultDesc: String? = null
}